lib/ace/editor.js

```diff
@@ -74,7 +75,13 @@ var Editor = function(renderer, session) {
     this.renderer = renderer;
 
     this.commands = new CommandManager(useragent.isMac ? "mac" : "win", defaultCommands);
-    this.textInput  = new TextInput(renderer.getTextAreaContainer(), this);
+    
+    var iOS = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;
```

Comment:
That is because MS tried to break browser detection - see http://stackoverflow.com/a/9039885/101970